packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1319,6 +1340,8 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
     return _layer;
   }
 
+  /// Whether this render object requires an update to its compositing state.
+  bool get needsCompositingBitsUpdate => _needsCompositingBitsUpdate;
```

Comment:
markNeedsCompositingBitsUpdate() was already public; it seemed reasonable to me to add a getter for state that you can already set.  But if you're bothered by this I can look into a different mechanism. There is already an example of using a semantics listener to validate state in a different test, it's just more verbose.